src/components/ConcentrationRisk/ConcentrationRisk.js

```diff
@@ -40,20 +41,28 @@ class ConcentrationRisk extends PureComponent {
   }
 
   parseData = (filteredData) => {
+    let allBalance = 0
     const summaryData = filteredData.reduce((acc, entry) => {
       const { currency, balanceUsd } = entry
       if (balanceUsd) {
         acc[currency] = (acc[currency] || 0) + balanceUsd
+        allBalance += balanceUsd
       }
       return acc
     }, {})
 
-    const tableData = _keys(summaryData).map(key => ({
-      currency: key,
-      balanceUsd: summaryData[key],
-    }))
+    const tableData = _sortBy(_keys(summaryData).map((key) => {
+      const balanceUsd = summaryData[key]
+      const percent = ((balanceUsd / allBalance) * 100)
+
+      return {
+        currency: key,
+        balanceUsd,
+        percent: fixedFloat(percent),
+      }
+    }), 'balanceUsd').reverse()
 
-    const chartData = tableData.map(({ currency, balanceUsd }) => ({
+    const chartData = tableData.filter(({ percent }) => percent > 0.1).map(({ currency, balanceUsd }) => ({
```

Comment:
can we drop the chain in 2 lines?
.filter
.map
for readability

Reply:
@jarindr changed :)